content/applications/websites/ecommerce/managing_products.rst

```diff
@@ -1,5 +1,3 @@
-:nosearch:
```

Comment:
Why?

Reply:
The toctree was literally copy/pasted from another toctree file. The question is why the others have :nosearch:? I'm fixing changing it

Reply:
The `:nosearch` metatag is intended. It's used to remove the page from search results when it contains no valuable information.

Reply:
I don't understand then, @AntoineVDV is saying it's for content with no valuable information, but @jcs-odoo is saying it's for category pages without any relevant content, which *is not* my case. Therefore, the content *is* relevant ?

Reply:
Here is what I mean:
![image](https://user-images.githubusercontent.com/15896813/219004542-f04e90a4-0951-4486-9073-4ffef5098e40.png)

Having **Cross-selling and upselling** in the search results is what we want. But there is no other valuable information for the reader than "you should have clicked the first result" on **Products** because it only contains a list of pages.

Reply:
Much clearer, thank you